docs/tasks/administer-cluster/access-cluster-api.md

```diff
@@ -168,17 +168,22 @@ There are [client libraries](/docs/reference/client-libraries/) for accessing th
 
 ### Accessing the API from a Pod
 
-When accessing the API from a pod, locating and authenticating
+When accessing the API from a Pod, locating and authenticating
 to the API server are somewhat different.
 
-The recommended way to locate the apiserver within the pod is with
-the `kubernetes` DNS name, which resolves to a Service IP which in turn
-will be routed to an apiserver.
+The easiest way to use the Kubernetes API from a Pod is to use
+one of the official [client libraries](/docs/reference/client-libraries/). These
+libraries can automatically discover the apiserver and authenticate automatically.
 
-The recommended way to authenticate to the apiserver is with a
-[service account](/docs/user-guide/service-accounts) credential.  By kube-system, a pod
+While running in a Pod, the Kubernetes apiserver is accessible via a Service named
+`kubernetes` in the `default` namespace. Therefore, Pods can use the 
```

Comment:
the apiserver -> the API server

Reply:
We currently don't have guidance on what it should be, hence my confusion. 
https://www.google.com/search?q=site%3Akubernetes.io+"apiserver"&oq=site%3Akubernetes.io+"apiserver"
`apiserver` is indeed a Kubernetes term and it refers to the `kube-apiserver` component, but in this case I think we're trying to say "the API server"

Reply:
done